scripts/squads.yml

```diff
@@ -192,6 +177,18 @@ squads:
       - *DanielH
       - *DanielS
       - *Diego
+      - *Javier
+  - name: Native Apps Platform
+    desc: Engineering work like tooling, CI and development processes.
+    members:
+      - *David
+      - *Ilya
+      - *Olivier
+  - name: Onboarding and Navigation
+    desc: General user experience in the app.
+    members:
+      - *Sergey
+      - *Yuri
```

Comment:
So instead of “Onboarding and navigation” we should have the following:
```
  - name: Activate
     desc: Onboard users so that they understand the value we provide.
     members:
       - *Sergey
  - name: Retain
     desc: Engage users with our key features in a dynamic, personalised manner.
     members:
       - *Yuri
```